src/ansys/grantami/recordlists/_models.py

```diff
@@ -285,6 +287,17 @@ def _to_model(self) -> models.GrantaServerApiListsDtoListItem:
         logger.debug(model.to_str())
         return model
 
+    def _to_delete_list_item_model(self) -> models.GrantaServerApiListsDtoDeleteRecordListItem:
+        """Generate the Delete List Item DTO for use with the auto-generated client code."""
+        logger.debug("Serializing RecordListItem to API model")
+        model = models.GrantaServerApiListsDtoDeleteRecordListItem(
+            database_guid=self.database_guid,
+            record_history_guid=self.record_history_guid,
+            record_version=self.record_version,
```

Comment:
There is no table_guid on the deletion model? Or is an item fully identified without it anyway?

Reply:
An item will be fully identified without it, since a record reference only needs a DB and record identifier. I'm not sure the rationale behind the asymmetry between the Create and Delete DTO, but it seems to work as expected